voorhoede/heads-up
# Feat/twitter label and data support

## What change(s) were made
- Added `twitter:label{1,2}`, `twitter:data{1,2}` to properties list.
- Added `twitter:label{1,2}`, `twitter:data{1,2}` to rich preview.

## How to test
- Visit any medium.com article and check the Slack tab in the browser extension.

## Related Trello ticket(s)
- [https://trello.com/c/ck6C3fzu](https://trello.com/c/ck6C3fzu)

## Checks
